packages/accordion/test/dom/__snapshots__/accordion-panel.test.snap.js

```diff
@@ -4,17 +4,17 @@ export const snapshots = {};
 snapshots["vaadin-accordion-panel host default"] = 
 `<vaadin-accordion-panel>
   <vaadin-accordion-heading
-    aria-controls="content-vaadin-accordion-panel-1"
-    id="summary-vaadin-accordion-panel-0"
+    aria-controls="content-vaadin-accordion-panel-0"
+    id="summary-vaadin-accordion-panel-1"
```

Comment:
ID differences are caused by the controllers init order, as the `ContentController` is moved to the mixin.